app/views/feedbacks/index.html.slim

```diff
@@ -0,0 +1,42 @@
+h1= t_title
+
+table.table.table-striped
+  thead
+    tr
+      th= t_model(Volunteer)
+      th= t_attr(:feedback_activity, Feedback)
+      th= t_attr(:goals, Feedback)
+      th= t_attr(:achievements, Feedback)
+      th= t_attr(:future, Feedback)
+      th= t_attr(:comments, Feedback)
+      th= t_attr(:conversation, Feedback)
+      - unless current_user.volunteer?
+        th= t_attr(:author, Feedback)
+      th colspan='3'
+
+  tbody
+    - @feedbacks.each do |record|
+      tr
+        td= link_to record.volunteer.full_name, volunteer_path(record.volunteer)
+        td
+          - if record.assignment?
+            = link_to record.feedbackable.to_label, client_path(record.feedbackable.client)
+          - else
+            = link_to record.feedbackable.to_label, group_offer_path(record.feedbackable)
+        td= record.goals
+        td= record.achievements
+        td= record.future
+        td= record.comments
+        td= t(record.conversation)
+        - unless current_user.volunteer?
+          td
+            - if record.author.id == record.volunteer.user.id
```

Comment:
You can also compare ActiveRecord objects directly (`record.author == record.volunteer.user`), by default it uses the `id` for comparison.